### ISSUE TITLE
x86 docker-compose script

### ISSUE THREAD

**Issue by dogi** [treehouses/moodole#10]
```
moodledb: 
  image: postgres 
  container_name: moodledb
  ports: 
    - "5432:5432" 
  environment: 
    - POSTGRES_DATABASE=moodle 
    - POSTGRES_USER=moodle 
    - POSTGRES_PASSWORD=moodle
moodle: 
  image: treehouses/moodle:x86
  container_name: moodle
  ports:
    - "8080:80"
  links:
    - moodledb
  volumes: 
    - /home/pi/apache2:/var/run/apache2
  environment: 
    - MOODLE_URL=http://moodoledemo.local:80
```

**Comment by empeje** [comment]
Please try also this dogi
```
version: '2'
services:
  moodledb:
    image: postgres
    container_name: moodledb
    ports:
    - "5432:5432"
    environment:
    - POSTGRES_DATABASE=moodle
    - POSTGRES_USER=moodle
    - POSTGRES_PASSWORD=moodle
  moodle:
    image: treehouses/moodle:x86-pgsql
    container_name: moodle
    ports:
    - "8080:80"
    volumes:
    - /home/pi/apache2:/var/run/apache2
    environment:
    - MOODLE_URL=http://moodle:80
```

**Comment by empeje** [comment]
This also solved with #12
